src/components/InputForm.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+	<div>
+
+		<div v-if="checkParam">{{ checkParam }}</div>
+
+		<p>
+			<label for="name">名前:<span class="required">(必須)</span>
+			</label>
+			<input type="text" name="name" v-model="formData.name" />
+		</p>
+		<p>
+			<label for="email">メールアドレス:<span class="required">(必須)</span>
+			</label>
+			<input type="email" name="email" v-model="formData.email" />
+		</p>
+		<p>
+			<label for="phone">電話番号:<span class="required">(必須)</span>
+			</label>
+			<input type="phone" name="phone" v-model="formData.phone" />
+		</p>
+		<p>
+			<label for="date">年月日：</label>
+			<input type="date" name="date" v-model="formData.date" />
+		</p>
+		<p>
+			<label for="zipCode">郵便番号：</label>
+			<input type="text" name="zipCode" v-model="formData.zipCode" />
+		</p>
+		<p>
+			<label for="prefecture">都道府県：</label>
+			<select v-model="formData.prefecture" id="prefecture" name="prefecture">
+				<option disabled value="">都道府県</option>
+				<option v-for="(pref, index) in prefectures" :key="index" :value="pref">
+					{{ pref }}
+				</option>
+			</select>
+		</p>
+		<p>
+			<label for="comment">コメント：</label>
+			<textarea name="comment" v-model="formData.comment"></textarea>
+		</p>
+		<button type="button" @click="handleSubmit">確認画面へ</button>
+	</div>
+</template>
+
+<script>
+export default {
+	name: 'InputForm',
+	props: {
+		defaultFormData: { Object, default: () => ({}) },
+	},
+	data() {
+		return {
+			formData: {
+				name: this.defaultFormData.name || '',
+				email: this.defaultFormData.email || '',
+				phone: this.defaultFormData.phone || '',
+				date: this.defaultFormData.date || '',
+				zipCode: this.defaultFormData.zipCode || '',
+				prefecture: this.defaultFormData.prefecture || '',
+				comment: this.defaultFormData.comment || ''
+			},
+			prefectures: [
+				'北海道','青森県','岩手県','宮城県','秋田県','山形県',
+				'福島県','茨城県','栃木県','群馬県','埼玉県','千葉県',
+				'東京都','神奈川県','新潟県','富山県','石川県','福井県',
+				'山梨県','長野県','岐阜県','静岡県','愛知県','三重県',
+				'滋賀県','京都府','大阪府','兵庫県','奈良県','和歌山県',
+				'鳥取県','島根県','岡山県','広島県','山口県','徳島県',
+				'香川県','愛媛県','高知県','福岡県','佐賀県','長崎県',
+				'熊本県','大分県','宮崎県','鹿児島県','沖縄県'
+			],
+			checkParam: '',
+		}
+	},
+	methods: {
+		handleSubmit() {
+			this.handleFormCheck();
+			if(!this.checkParam) {
+				this.$emit('onSubmit', this.formData)
+			}
+			return;
+		},
+		handleFormCheck() {
+			// 名前
+			if (!this.formData.name || this.formData.name.trim() === '') {

```

Comment:
```suggestion
			if (this.formData?.name.trim() === '') {
```
これでいけないですかね。なるべく冗長なコードは無くしたいので